src/components/select/select-textbox/select-textbox.component.js

```diff
@@ -13,11 +15,15 @@ const SelectTextbox = ({
   onClick,
   onFocus,
   onBlur,
+  onChange,
   selectedValue,
   required,
+  hasTextCursor,
```

Comment:
Would be good to have this defined in the prop table with `@ignore @private` so it documents what it does to anyone working on Select down the line